mibitracker/request_helpers.py

```diff
@@ -475,13 +475,16 @@ def image_id(self, run_name, point_name):
         ).json()
 
         try:
-            assert len(results) == 1
+            len_results = len(results)
```

Comment:
This'll work, although is there a need to keep the TypeError clause or can we go straight to checking the length?

Reply:
@rfinck I don't know what is actually returned by the API call. Where can I check that?

Reply:
You can make a request to our instance and see what you get back? (Also, please edit the name of this PR to have a brief description in addition to the ticket number.)

Reply:
@rfinckit seems we do not need TypeError.

Reply:
no one has ever called me rfinckit before ...

if we don't need the TypeError, please remove it and then we can merge this it.

Reply:
@rfinck done